Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV/Excel.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System.Data;
+using System.Globalization;
+using System.Text;
+using ExcelDataReader;
+
+namespace Frends.Excel.ConvertToCSV;
+public static class Excel
+{
+    /// <summary>
+    /// Converts Excel file to CSV. [Documentation](https://github.com/FrendsPlatform/Frends.Excel/tree/main/Frends.Excel.ConvertToCSV)
+    /// </summary>
+    /// <param name="input">Input configuration</param>
+    /// <param name="options">Input options</param>
+    /// <param name="cancellationToken"></param>
+    /// <returns>Result containing the converted CSV string.</returns>
+    /// <exception cref="Exception"></exception>
+    public static Result ConvertToCSV(Input input, Options options, CancellationToken cancellationToken)
```

Comment:
Maybe [Propertytab] should be added, so input and options should be on their own tabs task editor.

Reply:
Added